torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -181,6 +181,45 @@ static const std::vector<OperatorGeneratorArgs> opGenArgs{
           push(stack, start + index * step);
         },
         aliasAnalysisFromSchema()),
+    OperatorGeneratorArgs(
+        TORCH_SELECTIVE_SCHEMA("prim::_SetGlobalCtx(int ctx_no) -> None"),
+        [](Stack& stack) {
+          // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
+          int64_t ctx_no = 0;
+          TORCH_CHECK(
+              (ctx_no >= 0) && (ctx_no < 64),
+              "context position must be in [0...63], received %d",
+              ctx_no);
+          pop(stack, ctx_no);
+          at::globalContext().assignGlobalCtx(ctx_no, true);
+        },
+        aliasAnalysisFromSchema()),
```

Comment:
I think these should actually be `aliasAnalysisConservative()` because they have side effects. Otherwise, TorchScript might try to optimize by moving these ops around the graph.

I think it should be safe to leave `_GetGlobalCtx` as being `aliasAnalysisFromSchema()`

Reply:
Done